src/components/About/About.js

```diff
@@ -110,10 +116,12 @@ const About = (props) => {
           <label for="exampleInputEmail1">Decription</label>
           <div className={styles["form-group"]}>
             <textarea
+              required
               id="w3review"
```

Comment:
Why we are maintaining these id and name?

Reply:
Done removed these as we are not using them for any specific person instead, they are here by mistake (stupid me copy pasted the code from w3 school)